data/remnant/remnant 2 side missions.txt

```diff
@@ -144,3 +144,224 @@ mission "Remnant: Visit to Paradise 1"
 			label mainreturn
 			`	Taely leads you around the area for a few more hours, pointing out historical features, places where they resurrected trades and skills from before the advent of flight, and other interesting things. Eventually she heads back to the train station, which deposits you back in the main spaceport.`
 			`	"I hope you enjoyed the tour, <first>. It was interesting to see our history through fresh eyes." She smiles micheviously. "And I enjoyed the hoverboarding, too. Maybe we can do it again sometime." She pauses for a moment, focusing on something you cannot see. "Looks like duty calls. Embers guide your way, Captain <last>." She waves, and heads off into the crowd.`
+
+
+
+mission "Remnant: Shattered Light 1"
+	name "Shattered Light - Contact"
+	invisible
+	source
+		government "Remnant"
+	destination "Viminal"
+	to offer
+		has "event: remnant: cognizance calm"
+		# random < 30
+	npc "scan outfits"
+		government "Ember Waste"
+		personality uninterested waiting staying mute
+		system Caeculus
+		ship "Dromedary Ghost" "???"
+		dialog "This ship barely registers on your sensors, and you can clearly see through it. The Remnant will want to know about it."
+	on complete
+		conversation
+			`Looking around the Viminal spaceport, you manage to spot Taely using something that looks kind of like a small catapult to throw a large rock across the cavern. A nearby Merganser leaps into the air and chases after it. The ship pounces on the rock, vanishes from sight, and then reappears next to Taely and drops the rock next to her. Without looking away from the Merganser she waves you over.`
+			`	"Well met, <first>, how goes your travels, found anything interesting?`
```

Comment:
`how goes your travels` that sounds weird to me but maybe it's correct

Also you're having the Merganser behave like a dog? I love it x)

Reply:
Thank you! It's a rather fun and visible way to showcase a ship being decidedly not a ship. It might downplay the sentience of the ship a bit; but on the other hand, people play catch or a variety of other games that are basically the same thing, so it fits.